### DISCUSSION TITLE
table that flows horizontally

### DISCUSSION THREAD

**Discussion by daviderusso1984** [mantinedev/mantine#2123] [Feature requests]
In the mobile version the table does not have horizontal scrolling and therefore is not usable if you have many columns

**Comment by rtivital** [comment]
You can set table overflow on your side, see example – https://github.com/mantinedev/mantine/blob/master/docs/src/components/MdxPage/MdxProvider/DataTable/DataTable.tsx#L20:L31

**Reply by daviderusso1984** [reply]
TX

**Reply by daviderusso1984** [reply]
TX

**Reply by daviderusso1984** [reply]
TX

**Reply by Aman5502** [reply]
It no longer accessible

**Reply by rtivital** [reply]
https://mantine.dev/core/table/#scroll-container

**Reply by Aman5502** [reply]
![image](https://github.com/user-attachments/assets/d274855c-6101-4726-a970-0bb0de6c1483)
I have around 15 + column in my table but it add scroll into my page instead of the table , as u see in the image 

 <Paper
      shadow={withShadow ? "xs" : undefined}
      withBorder={withBorder}
      mt={"xs"}
    >
      {title && (
        <Text size="lg" fw={500} mb="md">
          {title}
        </Text>
      )}
      <Table.ScrollContainer  minWidth={1000} maxHeight={400}>
        <Table highlightOnHover withTableBorder >
          <Table.Thead >
            <Table.Tr bg={"gray.0"} >
              {columns.map((column) => (
                <Th
                  reversed={false}
                  sorted={false}
                  onSort={() => {}}
                  key={column.key}
                >
                  {column.label}
                </Th>
              ))}
            </Table.Tr>
          </Table.Thead>
          <Table.Tbody>
            {data.map((row, index) => (
              <Table.Tr key={index}>
                {columns.map((column) => (
                  <Table.Td
                    key={`${index}-${column.key}`}
                    onClick={() =>
                      column.type === "clickable"
                        ? onRowClick?.(row[column.key])
                        : null
                    }
                    className={
                      column.type === "clickable"
                        ? classes.clickableCell
                        : classes.tableTd
                    }
                  >
                    {row[column.key]}
                  </Table.Td>
                ))}
              </Table.Tr>
            ))}
          </Table.Tbody>
        </Table>
      </Table.ScrollContainer>
    </Paper>
